src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -27,11 +28,12 @@ const windSpeed = new Set(conditions.windSpeed)
 export const Superlatives: React.FunctionComponent = () => {
   const unitSystem = useUnitSystem()
 
-  const lastHour = anHourAgoRounded()
+  const backOff = 6
+  const backOffTime = calcAnyHourAgoRounded(backOff)
 
   return (
     <UsePlatforms>
-      {({ platforms }) => <ShowSuperlatives laterThan={lastHour} {...{ platforms, unitSystem }} />}
+      {({ platforms }) => <ShowSuperlatives laterThan={backOffTime} {...{ platforms, unitSystem }} />}
     </UsePlatforms>
```

Comment:
I think this is duplicating the time in the loops below (74 and 90).

Reply:
Good catch! Thanks. I'll refactor so theres no duplicate on like 66